_security/configuration/opensearch-keystore.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: OpenSearch keystore
+parent: Configuration
+nav_order: 50
+---
+
+# OpenSearch keystore
+
+`opensearch-keystore` is a utility script for managing the OpenSearch keystore. This keystore provides a secure method to store sensitive information, such as passwords and keys, used in an OpenSearch cluster. The script allows you to securely create, list, add, and remove settings. It is included inside the OpenSearch distribution. 
+
+## Usage
+
+In order to use the OpenSearch keystore script, you must have access to the file system in which the script resides. 
+
+To use OpenSearch keystore, open a terminal and use the following command syntax: 
+
+```
+opensearch-keystore [command] [options]
+```
+{% include copy.html %}
+
+## Commands
+
+OpenSearch keystore supports the following the commands: 
+
+- `create`: Initializes a new keystore. If a keystore already exists, this command will overwrite the existing keystore.
+- `list`: Lists all settings in the keystore.
```

Comment:
```suggestion
- `list`: Lists all keystore settings.
```

Reply:
IMO I think `Lists all settings in the keystore.` is the way to go here. i.e. You list all files in a directory and not list all directory files.